examples/angular/jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/tsconfig.json'
+    }
+  },
+  setupFilesAfterEnv: ['<rootDir>/setupJest.js'],
+  transform: {
+    '^.+\\.(ts|js|html)$': 'ts-jest',

```

Comment:
Use the default `babel-jest` rather than `ts-jest`

Reply:
Sorry about that, but i have no ideas about how to not use ts-jest here. The reason why I am using it is that it's much easier and requires no changes to use decorators(@Component, @Injectable and etc.) within angular code with it. I have tried to use babel, but even if I add @babel/plugin-proposal-decorators plugin it wouldn't help and start throwing errors that it's not able to resolve all parameters for DataService. Maybe I am missing something with babel configuration?

Reply:
@SimenB finally, I have replaced it but has no luck with local .babelrc config for angular, so add two plugins to babel.config.js. Maybe this is due to https://github.com/babel/babel-loader/issues/780